### PR TITLE
Encryption for local seed

### PR DESCRIPTION
**Pull request by dannydeezy**

### CODE REVIEW

**Comment by dannydeezy** [comment]
To test, you can run:
```
node -r dotenv/config scripts/encrypt-seed.js
```

Then once you generate the encrypted seed, add it to your `.env` and try running the bot. On startup it will perform decryption and address verification which means it worked

File: README.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ We recommend using Sparrow unless you're running an instance of Bitcoin Core. Fo
 10) The tabs on the left will turn blue.
 11) Go to the receive tab. You will need this address for .env file configuration.
 
+### Encrypting the wallet seed
+To encrypt the wallet seed run:
+```agsl
+node -r dotenv/config scripts/encrypt-seed.js
```
Comment (habibitcoin) [inline README.md:69]: Can we make this an npm command? Also is agsl a typo?

**Review by habibitcoin — CHANGES_REQUESTED** [review]
2 qq's @dannydeezy + nit 

1. Is there a way to do this w/o introducing prompts dependency
2. Is there a way for replit users to do this? I suppose they need to be comfortable running the npm commands to setup Telegram too anyway. So not a big deal

File: README.md (hunk 1 above)
Comment (dannydeezy) [reply README.md:69]: good call.
agsl is always auto-recommended by my editor, im not sure what it is lol

**Comment by dannydeezy** [comment]
> 2 qq's @dannydeezy + nit
> 
> 1. Is there a way to do this w/o introducing prompts dependency
> 2. Is there a way for replit users to do this? I suppose they need to be comfortable running the npm commands to setup Telegram too anyway. So not a big deal

i think prompts is really nice and could be useful for future things like bulk selling, so i think maybe we're good to kee pit

currently replit users need to run the command from Shell, which isn't ideal but yeah same as telegram bot. I think a future task could be to make Replit setup even easier. also I'm not sure we would recommend this encryption for replit users yet because deploys can't accept prompts, so they wouldnt have a way to decrypt the seed

**Review by habibitcoin — APPROVED** [review]
Very cool! Works like a charm.

**Pull request merged**